src/SFA.DAS.AANHub.Data/Repositories/CalendarEventsReadRepository.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.EntityFrameworkCore;
+using SFA.DAS.AANHub.Domain.Entities;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+
+namespace SFA.DAS.AANHub.Data.Repositories;
+internal class CalendarEventsReadRepository : ICalendarEventsReadRepository
+{
+    private readonly AanDataContext _aanDataContext;
+    public CalendarEventsReadRepository(AanDataContext aanDataContext) => _aanDataContext = aanDataContext;
+
+    public async Task<CalendarEvent?> GetCalendarEvent(Guid id) => 
+        await _aanDataContext
+            .CalendarEvents
+            .AsNoTracking()
+            .Where(m => m.Id == id)
+            .Include(x => x.Attendees).ThenInclude(x => x.Member)
```

Comment:
Since only live attendees should be included, this should probably be:
` .Include(x => x.Attendees.Where(a => a.IsActive)`

Reply:
Correct

Reply:
Done